ui/widgets/nuxeo-file.js

```diff
@@ -244,12 +244,16 @@ import { UploaderBehavior } from './nuxeo-uploader-behavior.js';
           this.push('value', {
             'upload-batch': this.batchId,
             'upload-fileId': i.toString(),
+            _name: this.files[i].name,
```

Comment:
The real issue is when using the `nuxeo-file` widget in the data-table for multi complex property editing. When opening the dialog to edit a row, we need to make sure that the `value` object returned by the `nuxeo-file` widget from the dialog, which is not persisted yet server-side, has the needed info, i.e. blob name and transient location (in the transient store used by the batch upload), to properly render back in the edited row's cell where there's another `nuxeo-file` widget in `readonly` mode.

Goal is to provide a fix which does not require an evolution of VD's generation for maintenance branches which would be the case for:
>we should consider sharing the same files property between the two instances,

Plus, I don't see how we could share a `files` object anyway because the only common scope between table rows and and edit dialog is `item` anw.

Reply:
>Since these are being set on the value, won't these then be sent back to the server

Yes, but they are marked as private and it's OK IMO